### DISCUSSION TITLE
Does axolotl support training on completions only?

### DISCUSSION THREAD

**Discussion by ibicdev** [axolotl-ai-cloud/axolotl#997] [Q&A] [answered]
The `trl` library has something like `DataCollatorForCompletionOnlyLM` (https://huggingface.co/docs/trl/sft_trainer#train-on-completions-only) to train the model on just the responses but not the prompts. I'm fairly new to axolotl and might have missed it.  Thanks.

**Comment by MatiasEtcheve** [comment, marked as answer]
There is a parameter in the config file called `train_on_inputs`. You can simply put it at `false` like [here](https://github.com/OpenAccess-AI-Collective/axolotl/blob/70b46ca4f45b9cec5ded9563a68bb08607b56a3c/examples/llama-2/lora.yml#L42).
